src/components/Header/render.ts

```diff
@@ -2,36 +2,121 @@ import createElement from '../../utils/createElement';
 import headerLogo from '../../../templates/logo.png';
 import isHTMLElement from '../../utils/isHTMLElement';
 
-const createHeaderContainer = () => {
-  const headerContainer = createElement('header');
+const createBannerContent = () => {
   const h1 = createElement('h1');
-  const headerLogoImage = createElement('img', { src: headerLogo, alt: 'MovieList' });
+  const headerLogoImage = createElement('img', {
+    src: headerLogo,
+    alt: 'MovieList',
+  });
   h1.appendChild(headerLogoImage);
-  headerContainer.appendChild(h1);
+
+  return h1;
+};
+
+const createHeaderBanner = () => {
+  const container = createElement('div', { className: 'header-banner-wrapper' });
+  const h1 = createBannerContent();
+
+  container.appendChild(h1);
+
+  return container;
+};
+
+const createHeaderContainer = () => {
+  const headerContainer = createElement('header');
 
   return headerContainer;
 };
 
-const createSearchButton = () => {
-  const searchButton = createElement('button', { className: 'search-button', textContent: '검색' });
+const createWebSearchButton = () => {
+  const searchButton = createElement('button', {
+    className: 'web-search-button web',
+    textContent: '검색',
+  });
+  return searchButton;
+};
+
+const createMobileSearchButton = () => {
+  const searchButton = createElement('button', {
+    className: 'mobile-search-button mobile',
+    textContent: '검색',
+  });
   return searchButton;
 };
 
-const createSearchBar = () => {
-  const searchBox = createElement('form', { className: 'search-form', id: 'searchForm' });
-  const input = createElement('input', { type: 'search', placeholder: '검색' });
-  const searchButton = createSearchButton();
-  searchBox.appendChild(input);
-  searchBox.appendChild(searchButton);
+const createMobileSubmitButton = () => {
+  const submitButton = createElement('button', {
+    className: 'mobile-submit-button',
+    textContent: '검색',
+    type: 'submit',
+  });
+  return submitButton;
+};
+
+/* eslint-disable max-lines-per-function */
+const createMobileSearchBarContent = () => {
+  const boxContainer = createElement('div', {
+    className: 'box-container',
+  });
+  const input = createElement('input', {
+    type: 'search mobile',
+    placeholder: '검색',
+    className: 'mobile-search-input',
+  });
+  const searchButton = createMobileSearchButton();
+  const mobileSubmitButton = createMobileSubmitButton();
+  boxContainer.appendChild(input);
+  boxContainer.appendChild(searchButton);
+  boxContainer.appendChild(mobileSubmitButton);
+
+  return boxContainer;
```

Comment:
```suggestion
  const boxContainer = createElement('div', {
    className: 'box-container',
  });
  const input = createElement('input', {
    type: 'search mobile',
    placeholder: '검색',
    className: 'mobile-search-input',
  });
  const searchButton = createMobileSearchButton();
  const mobileSubmitButton = createMobileSubmitButton();
  
  boxContainer.appendChild(input);
  boxContainer.appendChild(searchButton);
  boxContainer.appendChild(mobileSubmitButton);

  return boxContainer;
```

선언부 밑에 개행이 하나 더 있으면 좋겠네요!